File: CinemaNow/CinemaNow.Model/Reports/MovieReservationSeatCount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CinemaNow.Models.Reports
{
    public class MovieReservationSeatCount
    {
        public int MovieId { get; set; }
        public string MovieTitle { get; set; }
        public int ReservationSeatCount { get; set; }
    }
}

File: CinemaNow/UI/cinemanow_desktop/lib/models/movie_reservation_seat_count.dart
import 'package:json_annotation/json_annotation.dart';

part 'movie_reservation_seat_count.g.dart';

@JsonSerializable()
class MovieReservationSeatCount {
  final int movieId;
  final String movieTitle;
  final int reservationSeatCount;

  MovieReservationSeatCount({
    required this.movieId,
    required this.movieTitle,
    required this.reservationSeatCount,
  });

  factory MovieReservationSeatCount.fromJson(Map<String, dynamic> json) =>
      _$MovieReservationSeatCountFromJson(json);
  Map<String, dynamic> toJson() => _$MovieReservationSeatCountToJson(this);
}
